### ISSUE TITLE
Last Errors and Errors tab no populated using FeatherJS

### ISSUE THREAD

**Issue by aditya0711** [slanatech/swagger-stats#42]
swsAPIstats.js

Error thrown at line 513
    this.promClientMetrics.api_response_size_bytes.labels(req.method,req.sws.api_path,res.statusCode).observe( req.sws.res_clength ? req.sws.res_clength : 0 );

if(req.sws.res_clength) = null. if response.c_length = 0;

Propagated from all the way from response intercept.
    
*FIX*
this.promClientMetrics.api_response_size_bytes.labels(req.method,req.sws.api_path,res.statusCode).observe( req.sws.res_clength ? req.sws.res_clength : 0 );

added a ternary operator to fix the issue. 

Let me know if this is an actual bug, will submit a PR.

**Comment by aditya0711** [comment]
Actually, backtracked it to 
swsCoreStats.js line 211
``` if ("_contentLength" in res){
        resContentLength = res['_contentLength']; // doesn't check if it's null. Can add a check if res._contentLength is null, it will jump to taking the value from the header. 
    }else{
        // Try header
        if(res.hasHeader('content-length')) {
            resContentLength = res.getHeader('content-length');
        }
    }```

**Comment by sv2** [comment]
Thank you!

**Comment by sv2** [comment]
Addressed in v0.95.6
